include/hip/devicelib/half/half2_math.hh

```diff
@@ -29,11 +29,33 @@
 extern "C++" {
 
 extern __device__ api_half2 rint(api_half2 x);
+extern __device__ api_half2 exp(api_half2 x);
+extern __device__ api_half2 log(api_half2 x);
+extern __device__ api_half2 log2(api_half2 x);
+extern __device__ api_half2 log10(api_half2 x);
+extern __device__ api_half2 ceil(api_half2 x);
+extern __device__ api_half2 floor(api_half2 x);
+extern __device__ api_half2 trunc(api_half2 x);
 
+extern __device__ api_half2 sin(api_half2 x);
+extern __device__ api_half2 cos(api_half2 x);
+
+extern __device__ api_half2 sqrt(api_half2 x);
 }
 
 static inline __device__ api_half2 rint_2h(api_half2 x) { return rint(x); }
+static inline __device__ api_half2 exp_2h(api_half2 x) { return exp(x); }
+static inline __device__ api_half2 log_2h(api_half2 x) { return log(x); }
+static inline __device__ api_half2 log2_2h(api_half2 x) { return log2(x); }
+static inline __device__ api_half2 log10_2h(api_half2 x) { return log10(x); }
+static inline __device__ api_half2 ceil_2h(api_half2 x) { return ceil(x); }
+static inline __device__ api_half2 floor_2h(api_half2 x) { return floor(x); }
+static inline __device__ api_half2 trunc_2h(api_half2 x) { return trunc(x); }
+
+static inline __device__ api_half2 sin_2h(api_half2 x) { return sin(x); }
+static inline __device__ api_half2 cos_2h(api_half2 x) { return cos(x); }
 
+static inline __device__ api_half2 sqrt_2h(api_half2 x) { return ::sqrt(x); }
 
 //__device__ __half2 h2ceil ( const __half2 h )
 //__device__ __half2 h2cos ( const __half2 a )
```

Comment:
placeholders

Reply:
I'm looking at this: https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH____HALF__ARITHMETIC.html

and some of these are not part of the API.  Can you clarify where you these are coming from?